src/ExpressionParser.js

```diff
@@ -67,6 +67,7 @@ export class ExpressionParser extends Parser {
         parser.consts = {
             ...parser.consts,
             pi: Math.PI,
+            tau: 2*Math.PI,
```

Comment:
Lets add spaces before and after the asterisk to adhere to the code style of the project.

Reply:
:heavy_check_mark: